packages/okta-vue/test/e2e/harness/src/components/SessionTokenLogin.vue

```diff
@@ -28,7 +28,7 @@ export default {
         password: this.password
       })
       .then(res =>
-        this.$auth.loginRedirect({
+        this.$auth.loginRedirect('/protected', {
```

Comment:
To validate we are navigating to a custom path, we're passing in `/protected` here. Previously, we would hit `/`.